<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>886</width>
    <height>577</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QHBoxLayout" name="horizontalLayout_7">
    <item>
     <layout class="QVBoxLayout" name="listLayout">
      <item>
       <widget class="QGroupBox" name="groupBox">
        <property name="title">
         <string>Pen Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QLabel" name="label">
             <property name="text">
              <string>Pen Color</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="penColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QVBoxLayout" name="colorPickerLayout"/>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QLabel" name="label_2">
             <property name="text">
              <string>Pen Width : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSpinBox" name="penWidthSpinbox">
             <property name="minimum">
              <number>2</number>
             </property>
             <property name="maximum">
              <number>12</number>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <item>
            <widget class="QLabel" name="label_3">
             <property name="text">
              <string>Pen Style : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="penStyleCombobox"/>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QGroupBox" name="groupBox_2">
        <property name="title">
         <string>Brush Properties</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_2">
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QLabel" name="label_4">
             <property name="text">
              <string>Brush Color : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="DragButton" name="brushColorButton">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_5">
           <item>
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Brush Style : </string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QComboBox" name="brushStyleComboBox"/>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QGroupBox" name="groupBox_3">
           <property name="title">
            <string>Canvas Properties</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <item>
             <widget class="QCheckBox" name="showgridCheckbox">
              <property name="text">
               <string>Show Grid</string>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QPushButton" name="centerSceneButton">
              <property name="text">
               <string>Center Scene</string>
              </property>
             </widget>
            </item>
            <item>
             <layout class="QHBoxLayout" name="horizontalLayout_6">
              <item>
               <widget class="QLabel" name="label_6">
                <property name="text">
                 <string>Scene Background : </string>
                </property>
               </widget>
              </item>
              <item>
               <widget class="QPushButton" name="sceneBackgroundButton">
                <property name="text">
                 <string/>
                </property>
               </widget>
              </item>
             </layout>
            </item>
           </layout>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QVBoxLayout" name="viewLayout"/>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>886</width>
     <height>26</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionAdd_Image"/>
    <addaction name="actionQuit"/>
    <addaction name="actionSave"/>
    <addaction name="actionLoad"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionCopy"/>
    <addaction name="actionCut"/>
    <addaction name="actionPaste"/>
    <addaction name="separator"/>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
   </widget>
   <widget class="QMenu" name="menuTools">
    <property name="title">
     <string>Tools</string>
    </property>
    <addaction name="actionCursor"/>
    <addaction name="actionPen"/>
    <addaction name="actionEraser"/>
    <addaction name="actionEllipse"/>
    <addaction name="actionRectangle"/>
    <addaction name="actionStar"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuTools"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionSave"/>
   <addaction name="actionLoad"/>
   <addaction name="separator"/>
   <addaction name="actionCopy"/>
   <addaction name="actionCut"/>
   <addaction name="actionPaste"/>
   <addaction name="separator"/>
   <addaction name="actionCursor"/>
   <addaction name="actionPen"/>
   <addaction name="actionEraser"/>
   <addaction name="actionEllipse"/>
   <addaction name="actionStar"/>
   <addaction name="actionRectangle"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionAdd_Image">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Add Image</string>
   </property>
  </action>
  <action name="actionQuit">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/close.png</normaloff>:/images/close.png</iconset>
   </property>
   <property name="text">
    <string>Quit</string>
   </property>
  </action>
  <action name="actionPen">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/pen.png</normaloff>:/images/pen.png</iconset>
   </property>
   <property name="text">
    <string>Pen</string>
   </property>
  </action>
  <action name="actionEraser">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/eraser.png</normaloff>:/images/eraser.png</iconset>
   </property>
   <property name="text">
    <string>Eraser</string>
   </property>
  </action>
  <action name="actionEllipse">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/circle.png</normaloff>:/images/circle.png</iconset>
   </property>
   <property name="text">
    <string>Ellipse</string>
   </property>
  </action>
  <action name="actionRectangle">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/rectangle1.png</normaloff>:/images/rectangle1.png</iconset>
   </property>
   <property name="text">
    <string>Rectangle</string>
   </property>
  </action>
  <action name="actionStar">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/star.png</normaloff>:/images/star.png</iconset>
   </property>
   <property name="text">
    <string>Star</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionCursor">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/cursor.png</normaloff>:/images/cursor.png</iconset>
   </property>
   <property name="text">
    <string>Cursor</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/save.png</normaloff>:/images/save.png</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionLoad">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/open.png</normaloff>:/images/open.png</iconset>
   </property>
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/copyIcon.png</normaloff>:/images/copyIcon.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/cutIcon.png</normaloff>:/images/cutIcon.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/pasteIcon.png</normaloff>:/images/pasteIcon.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/undoIcon.png</normaloff>:/images/undoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resource.qrc">
     <normaloff>:/images/redoIcon.png</normaloff>:/images/redoIcon.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <customwidgets>
  <customwidget>
   <class>DragButton</class>
   <extends>QPushButton</extends>
   <header location="global">dragbutton.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="resource.qrc"/>
 </resources>
 <connections/>
</ui>
